browser/src/slideshow/transition/CombTransition.ts

```diff
@@ -1,3 +1,5 @@
+/** */
+
```

Comment:
@pedropintosilva  I didn't understand why you added `/** */` here. Is there any reason? Please explain to me.

Reply:
@codewithvk I was the one who added them, based on c37bbf69c5eba8e15a348cea03e0133029e804b4, please see the commit message there.

Reply:
Thanks @thebearon ! Nice work 🥇